src/03-control-flow.js

```diff
@@ -128,7 +179,19 @@ function aroundTheWorldGreetingWithSwitch(language) {}
  *  calculateLetterGrade(99);
  *  //> "A"
  */
-function calculateLetterGrade(grade) {}
+function calculateLetterGrade(grade) {
+  if (grade < 60){
+    return "F";
+  } else if (60 <= grade && grade < 70){
+    return "D";
+  } else if (70 <= grade && grade < 80){
+    return "C";
+  } else if (80 <= grade && grade < 90){
+    return "B";
+  } else if (grade >= 90){
+    return "A";
```

Comment:
These last two functions are excellent.  Once again just make sure to space out our opening. ` {  `.  These are super specific criticisms but if we get these habits down now you'll be in a great place in a couple months.